haproxy_exporter.go

```diff
@@ -187,15 +191,17 @@ var (
 		61: newBackendMetric("http_total_time_average_seconds", "Avg. HTTP total time for last 1024 successful connections.", nil),
 	}
 
-	haproxyUp = prometheus.NewDesc(prometheus.BuildFQName(namespace, "", "up"), "Was the last scrape of haproxy successful.", nil, nil)
+	haproxyInfo = prometheus.NewDesc(prometheus.BuildFQName(namespace, "version", "info"), "haproxy version info.", []string{"release_date", "version"}, nil)
```

Comment:
What about using the official spelling? I had a comment about the typo as well before ;-)

```suggestion
	haproxyInfo = prometheus.NewDesc(prometheus.BuildFQName(namespace, "version", "info"), "HAProxy version info.", []string{"release_date", "version"}, nil)
```